packages/shared/src/components/search/SearchResults/SearchResultsTags.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from 'react';
+import { WidgetCard } from '../../widgets/WidgetCard';
+import { TagLink } from '../../TagLinks';
+import { ListItemPlaceholder } from '../../widgets/ListItemPlaceholder';
+import { SearchSuggestion } from '../../../graphql/search';
+
+interface SearchResultsTagsProps {
+  items: string[];
+  isLoading: boolean;
+  onTagClick: (suggestion: SearchSuggestion) => void;
+}
+
+export const SearchResultsTags = (
+  props: SearchResultsTagsProps,
+): ReactElement => {
+  const { items = [], isLoading, onTagClick } = props;
+  return (
+    <WidgetCard heading="Related tags">
+      {!!items.length && (
```

Comment:
If you pass `null` on the `items` prop, it will not fallback to `[]`.
```suggestion
      {!!items?.length && (
```